addon/utils/fd-activity-diagram-primitives.js

```diff
@@ -1,7 +1,7 @@
 import joint from 'npm:jointjs';
 import './fd-common-primitives';
 
-joint.shapes.flexberryUml.BaseObject.define('flexberryUml.SignalReceiptRight', {
+joint.shapes.flexberry.uml.BaseObject.define('flexberry.uml.SignalReceiptRight', {
```

Comment:
Use the import for `BaseObject` instead of the full `joint.shapes.flexberry.uml` namespace.

Reply:
In future, everything will be moved to Objects and this file will be removed.